src/components/stickyFooter/StickyFooter.tsx

```diff
@@ -0,0 +1,48 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+import {IClassName} from '../../utils/ClassNameUtils';
+import {SideNavigation} from '../sideNavigation/SideNavigation';
+import * as styles from './StickyFooter.scss';
+
+export interface IStickyFooterProps {
+    classes?: IClassName;
+}
+
+export interface IStickyFooterState {
+    withSideNav?: boolean;
+}
+
+export class StickyFooter extends React.Component<IStickyFooterProps, IStickyFooterState> {
+    static ID = 'StickyFooter';
+    static withSideNavClass = styles.stickyFooterWithSideNavClass;
+
+    constructor(props: IStickyFooterProps, state: IStickyFooterProps) {
+        super(props, state);
+        this.state = {};
+    }
+
+    componentDidMount() {
+        this.setState({withSideNav: !!document.querySelector('.navigation-wrapper-opened')});

```

Comment:
This should probably be the initial state in the constructor

Reply:
i think it could _potentially_ cause an undesired state of the class instantiating before the sidenav is in the dom, I vaguely remember this happening when i was testing

Reply:
removed logic